main.cpp

```diff
@@ -57,9 +62,104 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
     
-    
+    short int shortInt01 = 1;
+    short int shortInt02 = -22;
+    short int shortInt03 = 1231;
+
+    unsigned short int uShortInt01 = 0;
+    unsigned short int uShortInt02 = 34000;
+    unsigned short int uShortInt03 = 65535;
+
+    unsigned int uInt01 = 0;
+    unsigned int uInt02 = 65644;
+    unsigned int uInt03 = 4294967295;
+
+    int int01 = -2147483648;
+    int int02 = 0;
+    int int03 = 2147483647;
+
+    long int longInt01 = -2147483648;
+    long int longInt02 = 0;
+    long int longInt03 = 2147483647;
+
+    unsigned long int uLongInt01 = 123123;
+    unsigned long int uLongInt02 = 0;
+    unsigned long int uLongInt03 = 2147483647;
+
+    // long long int longLongInt01 = -2147483648;
+    // long long int longLongInt02 = 0;
+    // long long int longLongInt03 = 2147483647;
+
+    // unsigned long long int uLongLongInt01 = 1231231;
+    // unsigned long long int uLongLongInt02 = 0;
+    // unsigned long long int uLongLongInt03 = 2147483647;
+
+    signed char sChar01 = -100;
+    signed char sChar02 = 0;
+    signed char sChar03 = 100;
+
+    unsigned char uChar01 = 12;
+    unsigned char uChar02 = 0;
+    unsigned char uChar03 = 121;
+
+    float float01 = -1.0f;
+    float float02 = 2.5f;
+    float float03 = 2.2f;
+
+    double double01 = -2147483648;
+    double double02 = 0;
+    double double03 = 2147483647;
+
+    long double longDouble01 = -2147483648;
+    long double longDouble02 = 0;
+    long double longDouble03 = 2147483647;            
     
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(shortInt01);
+    ignoreUnused(shortInt02);
+    ignoreUnused(shortInt03);
+    ignoreUnused(uShortInt01);
+    ignoreUnused(uShortInt02);
+    ignoreUnused(uShortInt03);
+    ignoreUnused(uInt01);
+    ignoreUnused(uInt02);
+    ignoreUnused(uInt03);
+    ignoreUnused(int01);
+    ignoreUnused(int02);
+    ignoreUnused(int03);
+    ignoreUnused(longInt01);
+    ignoreUnused(longInt02);
+    ignoreUnused(longInt03);
+    ignoreUnused(uLongInt01);
+    ignoreUnused(uLongInt02);
+    ignoreUnused(uLongInt03);
+    // ignoreUnused(longLongInt01);
+    // ignoreUnused(longLongInt02);
+    // ignoreUnused(longLongInt03);
+    // ignoreUnused(uLongLongInt01);
+    // ignoreUnused(uLongLongInt02);
+    // ignoreUnused(uLongLongInt03);
+    ignoreUnused(sChar01);
+    ignoreUnused(sChar02);
+    ignoreUnused(sChar03);
+    ignoreUnused(uChar01);
+    ignoreUnused(uChar02);
+    ignoreUnused(uChar03);
+    ignoreUnused(float01);
+    ignoreUnused(float02);
+    ignoreUnused(float03);
+    ignoreUnused(double01);
+    ignoreUnused(double02);
+    ignoreUnused(double03);
+    ignoreUnused(longDouble01);
+    ignoreUnused(longDouble02);
+    ignoreUnused(longDouble03);
```

Comment:
You could have passed all of them at the same time to `ignoreUnused()`.  The example function below demonstrates this.

Reply:
OK